packages/verification-vue-component/src/style/components/_buttons.scss

```diff
@@ -0,0 +1,31 @@
+.btn {
+    padding: 8px 10px;
+    font-size: $font-size-medium;
+    font-weight: 500;
+    border-radius: 3px;
+    border: none;
+    outline: none;
+
+    &:active {
+        outline: none;
+    }
+
+    &.primary {
+        background-color: $verification-blue;
+        color: $white;
+    }
+
+    &.secondary {
+        background-color: #EEF3FA;
+        color: $verification-blue;
+    }
+
+    &.light {
+        background-color: white;
+        color: #0669f2;
+    }
+
+    &:nth-child(2) {
```

Comment:
this feels quite general here for a generic scss like `_buttons.scss`. If `_buttons.scss` is imported somewhere else all second buttons would have the margin

Reply:
You are right, extended it now so all buttons except the first has margin-left.

Reply:
This style would apply to all buttons, don't you think it'd be better to add a additional class to this "div"-element : ` <div class="actions navigate">` and just set the margin for this the second button inside this element

Reply:
I think it's correct to define it in general. As soon as there are multiple buttons next to each other they need to have some space in between.

Reply:
fair enough 👌